diwako_cbrn.vr/scripts/cbrn/functions/fn_detectorGeigerPFH.sqf

```diff
@@ -0,0 +1,45 @@
+_low_rad = ["cbrn_low_rad_0", "cbrn_low_rad_1", "cbrn_low_rad_2", "cbrn_low_rad_3"];
+_mid_rad = ["cbrn_mid_rad_0", "cbrn_mid_rad_1", "cbrn_mid_rad_2", "cbrn_mid_rad_3"];
+_high_rad = ["cbrn_high_rad_0", "cbrn_high_rad_1", "cbrn_high_rad_2", "cbrn_high_rad_3"];
+_danger_rad = ["cbrn_danger_rad_0", "cbrn_danger_rad_1", "cbrn_danger_rad_2", "cbrn_danger_rad_3"];
+_deadly_rad = ["cbrn_deadly_rad_0", "cbrn_deadly_rad_1", "cbrn_deadly_rad_2", "cbrn_deadly_rad_3"];
+_run_rad = ["cbrn_run_rad_0", "cbrn_run_rad_1", "cbrn_run_rad_2", "cbrn_run_rad_3"];
+_omg_rad = ["cbrn_omg_rad_0", "cbrn_omg_rad_1", "cbrn_omg_rad_2", "cbrn_omg_rad_3"];
```

Comment:
These could be preprocessor macros. So you do not need to define them every time this code is run.

Reply:
Obsolete since we're switching "back" to the 2 second clips